app/assets/javascripts/search.js

```diff
@@ -13,7 +13,11 @@ function search() {
         var searchTitle = title.indexOf(searchInput) >= 0;
         var searchBody  = body.indexOf(searchInput) >= 0;
 
-        (searchTitle || searchBody) ? $(this).show() : $(this).hide();
```

Comment:
jshint will complain about this - but if you actually like using a ternery like this, you can configure jshint to ignore it by creating/adding a line to your .jshintrc file.